tensorflow_datasets/core/dataset_builder.py

```diff
@@ -205,6 +205,8 @@ def __init__(self, data_dir=None, config=None, version=None):
     else:  # Use the code version (do not restore data)
       self.info.initialize_from_bucket()
 
+    self._code_dir = os.path.dirname(inspect.getfile(self.__class__))
```

Comment:
Thank you. Would it be possible to add a test (create a dummy dataset with dummy checksums and dummy fake data) to make sure this works properly ?